assets/src/js/cookies.js

```diff
@@ -97,26 +97,22 @@ export const setupCookies = () => {
     }
   };
 
-  // Make the necessary cookies checked by default on user's first visit.
-  // Here if the No cookies set(absence of 'greenpeace' & 'no_track' cookies) consider as first visit of user.
+  // Make the necessary cookies checked by default on user's first visit, also create cookie to store last visited nro.
+  // Here if the No cookies set (absence of 'greenpeace' & 'no_track' cookies) consider as first visit of user.
   if (greenpeace === null && noTrack === null) {
     createCookie('greenpeace', ONLY_NECESSARY, 365);
+    createCookie('gp_nro', nro, 30);
```

Comment:
I'm realizing with this that our cookies are domain and not path specific, so if you visited `/luxembourg` first those cookies are set, and if you then go to `/mexico` `gp_nro` won't change.
I think we need to put this line out of any condition so it stays accurate, keeping its current behavior of changing its content when you visit a different nro.

Reply:
Good point 🤔 Maybe we could check the value of `gp_nro` and if we're on a different NRO site then we update it?

Reply:
Yep should work too :+1: